frontend/src/libs/components/input/input.tsx

```diff
@@ -48,10 +52,12 @@ const Input = <T extends FieldValues>({
       >
         {label}
       </span>
+      {icon && <Icon iconName={icon} className={styles['search-input-icon']} />}
```

Comment:
What if passed icon is not a `search` Icon? why do we need to specify the className with this relation?